### PR TITLE
Added params with detail for module:status command

### PR DESCRIPTION
**Pull request by chandru-199**
## Purpose of this pull request

This pull request (PR) - Added params with details for module:status command.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ In addition to the command arguments discussed here, see [Common arguments]({{ p
...
+
+Use the following command to list enabled and disabled modules:
+
+```bash
+bin/magento module:status <module-name> [--enabled] [--disabled]
+```
+
+where
+
+*  `<module-name>` is the name of the module for which you need to check the status.
```
Comment (keharper) [inline src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md:28]: Move this to last.
```suggestion
*  `<module-name>`specifies which module to return the status of. You cannot specify multiple module names.
```

File: src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ In addition to the command arguments discussed here, see [Common arguments]({{ p
...
+*  `--enabled` to list all the enabled modules.
+*  `--disabled` to list all the disabled modules.
```
Comment (keharper) [inline src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md:30]:
```suggestion
*  `--disabled` lists all disabled modules.
```

File: src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ In addition to the command arguments discussed here, see [Common arguments]({{ p
...
+Use the following command to list enabled and disabled modules:
+
+```bash
+bin/magento module:status <module-name> [--enabled] [--disabled]
+```
+
+where
+
+*  `<module-name>` is the name of the module for which you need to check the status.
+*  `--enabled` to list all the enabled modules.
```
Comment (keharper) [inline src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md:29]:
```suggestion
*  `--enabled` lists all enabled modules.
```

**Comment by chandru-199** [comment]
@keharper : Changes made. Please review.

**Comment by keharper** [comment]
running tests

**Pull request merged**